### PR TITLE
Set up Turbo repo and apps

### PR DESCRIPTION
**Pull request by Armfath**
- set up default Turbo monorepo config
- create NestJs app (default configs)
- create NextJs app (default configs)

### CODE REVIEW

**Comment by BOCOVO** [comment]
@Armfath 
The PR does not have description

File: apps/api/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    'plugin:prettier/recommended',
+  ],
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
```
Comment (BOCOVO) [inline apps/api/.eslintrc.js:20]: @Armfath 
Why are these rules disabled ?

File: apps/api/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (BOCOVO) [inline apps/api/tsconfig.json:1]: @Armfath 
Is this tsconfig extends the tsconfig package in the workspace ?

**Review by BOCOVO — COMMENTED** [review]
Some questions ?

File: apps/api/tsconfig.json (hunk 2 above)
Comment (Armfath) [reply apps/api/tsconfig.json:1]: No I keep it as default, we'll which configs to use and which to share

File: apps/api/.eslintrc.js (hunk 1 above)
Comment (Armfath) [reply apps/api/.eslintrc.js:20]: It is default from `$ nest new`

**Comment by Armfath** [comment]
> @Armfath The PR does not have description

@BOCOVO  Added

**Review by BOCOVO — APPROVED** [review]

**Pull request merged**